build/scripts/build.ps1

```diff
@@ -397,45 +516,38 @@ try {
         Redirect-Temp
     }
 
+    if ($isAnyTestSpecial) {
+        Test-Special
+        exit 0
+    }
+
     if ($restore) { 
         Write-Host "Running restore"
         Restore-All -msbuildDir $msbuildDir 
     }
 
-    if ($testBuildCorrectness) {
-        Exec-Block { & ".\build\scripts\test-build-correctness.ps1" -config $buildConfiguration } | Out-Host
-        exit 0
-    }
-
     if ($bootstrap) {
-        Write-Host "Making bootstrap"
         $bootstrapDir = Make-BootstrapBuild
     }
 
-    if ($testDeterminism) {
-        Exec-Block { & ".\build\scripts\test-determinism.ps1" -bootstrapDir $bootstrapDir } | Out-Host
-        exit 0
+    if ($build) {
+        Build-Artifacts
     }
 
-    if ($testPerfCorrectness) {
-        Test-PerfCorrectness
-        exit 0
+    if ($sign) {
+        Run-SignTool
     }
 
-    if ($testPerfRun) {
-        Test-PerfRun
-        exit 0
+    # Must come after signing so that only the signed binaries are packed. Unlike 
+    # VSIX, NuGet doesn't support re-packing hence we have to order it this way.
```

Comment:
Comment seems odd: why is NuGet different than VSIX here? They're both OPC packages...?